commands/upload.ts

```diff
@@ -157,20 +156,20 @@ exports.handler = async options => {
       return;
     }
     upload(
-      accountId,
+      account,
       absoluteSrcPath,
       normalizedDest,
       getFileMapperQueryValues(mode, options)
     )
       .then(() => {
         logger.success(
           i18n(`${i18nKey}.success.fileUploaded`, {
-            accountId,
+            account,
```

Comment:
Another one that should be `accountId: account`. There are a few other `i18n`s and `ApiErrorContexts` in this file as well that need to be updated

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1212/commits/11c8c119479c484027f71dffa1ad97d64951d2d8